### DISCUSSION TITLE
Unable to claim the azure student pack

### DISCUSSION THREAD

**Discussion by Dhivakarkumar8** [community/community#41197] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

While I am trying to claim the free azure student pack it's shows your are not eligible or sometimes it's shows unable to verify your university I'd.
Any solutions to solve this one

**Comment by queenofcorgis** [comment]
Hi @Dhivakarkumar8,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this topic, perhaps [Azure Education Hub](https://aka.ms/startEDU). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
